appinventor/appengine/src/com/google/appinventor/client/youngandroid/TextValidators.java

```diff
@@ -77,6 +77,7 @@ public static boolean checkNewProjectName(String projectName) {
   public static boolean checkNewComponentName(String componentName) {
 
     // Check that it meets the formatting requirements.
+    
```

Comment:
Revert the changes in this file. They are not relevant to what you're trying to accomplish in this PR.

Reply:
Fixed